packages/eas-cli/src/ora.ts

```diff
@@ -0,0 +1,74 @@
+import oraReal from 'ora';
+
+import Log from './log';
+
+/**
+ * A custom ora spinner that sends the stream to stdout in CI, or non-TTY, instead of stderr (the default).
+ *
+ * @param options
+ * @returns
+ */
+export function ora(options?: oraReal.Options | string): oraReal.Ora {
+  const inputOptions = typeof options === 'string' ? { text: options } : options || {};
```

Comment:
- Add parentheses around `options || {}`. It's helpful for readers so they don't need to think about which operator has higher precedence. 
- Prefer `?? {}` over `|| {}`.